### PR TITLE
Rspec Model Status

### PR DESCRIPTION
**Pull request by ManhLeDuc**
## WHAT (optional)
- Rspec 100% coverage Model Status

## Evidence (Screenshot or Video)
- 
image
image
image

### CODE REVIEW

**Comment by ManhLeDuc** [comment]
ready

File: app/models/status.rb [hunk 1]
```diff
@@ -21,9 +21,13 @@ class Status < ApplicationRecord
   scope :subjects_ordered, ->{subject_type.ordered}
   scope :tasks_subject_id, ->(id){includes(:task).subject_id id}
 
-  def subject_finished_rate
+  def finished_rate
     if finishable_type == "Task"
-      errors.add :finishable_type, :not_permitted_type
+      if finished?
```
Comment (ngocvt-0484) [inline app/models/status.rb]:
```suggestion
      finished? ? Settings.complete_rate : 0
```

**Comment by ManhLeDuc** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**